examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_2.ipynb

```diff
@@ -0,0 +1,339 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Expect pandas to take >3 minutes on EC2, longer locally\n",
+    "\n",
+    "This is a good time to chat with your neighbor\n",
+    "Dicussion topics\n",
+    "- Do you work with a large amount of data daily?\n",
+    "- How big is your data?\n",
+    "- What’s the common use case of your data?\n",
+    "- Do you use any big data analytics tools?\n",
+    "- Do you use any interactive analytics tool?\n",
+    "- What’s are some drawbacks of your current interative analytic tools today?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `modin.pandas.read_csv`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "modin_df = pd.read_csv(s3_path, parse_dates=[\"tpep_pickup_datetime\", \"tpep_dropoff_datetime\"])\n",
+    "\n",
+    "end = time.time()\n",
+    "modin_duration = end - start\n",
+    "print(\"Time to read with Modin: {} seconds\".format(round(modin_duration, 3)))\n",
+    "\n",
+    "print(\"Modin is {}x faster than pandas at `read_csv`!\".format(round(pandas_duration / modin_duration, 2)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Are they equals?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "modin_df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Concept for exercise: Reduces\n",
+    "\n",
+    "In pandas, a reduce would be something along the lines of a `sum` or `count`. It computes some summary statistics about the rows or columns. We will be using `sum`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "pandas_sum = (pandas_df[:500000]).sum()\n",
+    "\n",
+    "end = time.time()\n",
+    "pandas_duration = end - start\n",
+    "\n",
+    "print(\"Time to count with pandas: {} seconds\".format(round(pandas_duration, 3)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "modin_sum = (modin_df[:500000]).sum()\n",
+    "\n",
+    "end = time.time()\n",
+    "modin_duration = end - start\n",
+    "print(\"Time to count with Modin: {} seconds\".format(round(modin_duration, 3)))\n",
+    "\n",
+    "print(\"Modin is {}x faster than pandas at `sum`!\".format(round(pandas_duration / modin_duration, 2)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Are they equals?"
+   ]
```

Comment:
the frames actually are not equal...

OmniSci just discards non-numeric columns in `.sum()` when pandas tries to use everything (for instance, concatenate strings):
<details><summary>Results under spoiler</summary>

```python
>>> pandas_sum
VendorID                                                          840127.0
passenger_count                                                   708874.0
trip_distance                                                   1369963.72
RatecodeID                                                        520400.0
store_and_fwd_flag       NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN...
PULocationID                                                      82640256
DOLocationID                                                      81309448
payment_type                                                      647593.0
fare_amount                                                     5674208.76
extra                                                             482188.3
mta_tax                                                           246220.5
tip_amount                                                       975269.91
tolls_amount                                                     109197.28
improvement_surcharge                                             148301.7
total_amount                                                    8389976.75
congestion_surcharge                                             1104077.5
dtype: object
>>> modin_sum
VendorID                 8.401270e+05
passenger_count          7.088740e+05
trip_distance            1.369964e+06
RatecodeID               5.204000e+05
PULocationID             8.264026e+07
DOLocationID             8.130945e+07
payment_type             6.475930e+05
fare_amount              5.674209e+06
extra                    4.821883e+05
mta_tax                  2.462205e+05
tip_amount               9.752699e+05
tolls_amount             1.091973e+05
improvement_surcharge    1.483017e+05
total_amount             8.389977e+06
congestion_surcharge     1.104078e+06
dtype: float64
```

</details>

Modin's result missing "store_and_fwd_flag" column as it's a string column, it's clearly seen in the output and would confuse users.

We should either:
1. Select only couple of columns to do the summation.
2. Select only numerical column by `df.select_dtypes(include="number").sum()`
3. Pass `numeric_only=True` option to `.sum()` method

Reply:
I am okay with 3rd option because we should show a speed up as well.

Reply:
@YarShev @dchigarev all 3 option aren't speed up

Reply:
indeed, when doing a fair comparison (when both frames sum only numerical columns) pandas appear to be faster even if we don't limit the number of rows to 500_000...

I briefly checked other reduction functions and they're all slower in Modin as well...

The bottleneck here is importing data to omnisci, if I forcibly trigger the import prior benchmarking then modin becomes much faster than pandas (but we can't do that in a real flow).

So it seems for now we should remove the cell, raise an appropriate issue, and get back to this in a separate PR when we come up with some solution.

Reply:
@YarShev @dchigarev I deleted this function